### ISSUE TITLE
RichTextBox shows some exceptions

### ISSUE THREAD

**Issue by ContainsK** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#1662]
When entering manually, the spacing is too large, and scrollbar displays abnormally

![image](https://user-images.githubusercontent.com/25956110/74620046-b3a0c180-5172-11ea-9ad9-fcb1b86b9a65.png)

**Comment by ContainsK** [comment]
@Keboo 
Is there a way to fix this problem quickly? Only on my local side, because I need it very much. Thank you!

**Comment by jespersh** [comment]
If you're talking about the paragraph spacing:
```xaml
<RichTextBox>
  <RichTextBox.Resources>
    <Style TargetType="{x:Type Paragraph}">
      <Setter Property="Margin" Value="0"/>
    </Style>
  </RichTextBox.Resources>
</RichTextBox>
```

**Comment by Keboo** [comment]
Fixed by #1680
